packages/user-interface/src/pages/case/case-page.tsx

```diff
@@ -65,6 +100,7 @@ const CasePage: FC<CasePageProps> = ({
     return array;
   }, [zaak]);
 
+  // TODO: please useMemo
```

Comment:
Good point, I will fix this in the next PR.

Reply:
Fixed it :)